datamade/la-metro-councilmatic
# Add Board Report Status definitions to About page

## Overview

See title.

### Checklist


### Demo
<img width="1437" alt="Screen Shot 2021-10-25 at 2 54 07 PM" src="https://user-images.githubusercontent.com/48227264/138761303-f9256825-6e5d-4956-8e47-65506fe08418.png">

## Testing Instructions

 * Check out this PR locally, bring up the app, and visit the About page
 * Make sure the status names and definitions make sense with each other & the table is responsive.

Handles #123
